avaframe/ana4Stats/probAnaCfg.ini

```diff
@@ -31,14 +31,15 @@ defaultComModuleCfg = True
 # list of parameters that shall be varied - separated by |
 varParList = relTh|mu
 
-variationType options: percent, range
+variationType options: percent, range, rangefromci
 # if percent: +- varParVariation percent of varParValue in varParSteps
 # if range: +- varParVariation of varParValue in varParSteps
+# if rangeFromCi: +- varParVariation but values taken from shape file ci value (only availabe for relTh, entTh, secondaryRelTh)
```

Comment:
There's a mismatch to line 39 rangeFromCi vs rangefromci

Reply:
set to rangefromci in both cases